packages/babel-plugin-transform-class-properties/test/fixtures/regression/T2983/actual.js

```diff
@@ -0,0 +1,7 @@
+call(class {
+  static test = true
+});
+
+export default class {
+  static test = true
+};
```

Comment:
Maybe I would add tests for named classes?

`class A` for both examples above?